framework/db/Command.php

```diff
@@ -780,6 +780,16 @@ public function execute()
 
             $this->pdoStatement->execute();
             $n = $this->pdoStatement->rowCount();
+            
+            // https://bugs.php.net/bug.php?id=61613
+            // Make sure that supported drivers throw an exception in case a statement of a multi-query failed
+            try {
+                while ($this->pdoStatement->nextRowSet()) {}
+            } catch (\Exception $e) {
+                if (strpos($e->getMessage(), 'SQLSTATE[IM001]') === false) {
```

Comment:
No, method always exists it's part of abstraction layer (`PDOStatement` class). It throws when method can not be used because underlying driver does not support this functionality.
Also see: https://github.com/php/php-src/blob/0437aa2abf1eb494e4555d7d4dfbdbf566a59bdc/ext/pdo/pdo_sqlstate.c#L285
If you google error code you see it relates to pdo_mysql, pdo_mssql etc...